source/cloud/cloud-administration/sso-gitlab.rst

```diff
@@ -1,40 +1,56 @@
 GitLab Single Sign-On
 =====================
 
-Configure Mattermost to use GitLab as a Single Sign-on (SSO) service for team creation, account creation, and user sign-in.
+Migrating from OAuth 2.0 to OpenID Connect
+-------------------------------------------
+
+OAuth 2.0 is being deprecated and replaced by OpenID Connect. Refer to product documentation to `convert your existing OAuth configuration <https://docs.mattermost.com/cloud/cloud-administration/converting-oauth-2.0-to-openid-connect>`__ for GitLab to the new OpenID Connect standard. 
```

Comment:
This is ok for cloud. But I just want to make sure for the on-premise edition, the TE will continue to allow gitlab via OAUTH.